tests/std/tests/GH_000639_nvcc_include_all/env.lst

```diff
@@ -5,3 +5,8 @@ PM_COMPILER="nvcc" PM_CL="--x cu -Xcompiler -Od,-EHsc,-nologo,-W4,-WX,-openmp"
 RUNALL_CROSSLIST
 PM_CL="-Xcompiler -MT"
 PM_CL="--debug -Xcompiler -MTd"
+RUNALL_CROSSLIST
+PM_CL=""

```

Comment:
Is no language mode different from each of the language modes?

Reply:
no language mode is the same as `-std=c++14`.

Reply:
If we're explicitly testing `-std=c++14` on the line below, then we shouldn't have an empty set here - that's redundant. (Elsewhere, our configs either use the pattern "nothing vs. something" or "explicitly specify all modes", and we use the latter for Standard modes.)